lib/grape/validations/params_scope.rb

```diff
@@ -23,13 +25,14 @@ class ParamsScope
       #   validate if this param is present in the parent scope
       # @yield the instance context, open for parameter definitions
       def initialize(opts, &block)
-        @element      = opts[:element]
-        @parent       = opts[:parent]
-        @api          = opts[:api]
-        @optional     = opts[:optional] || false
-        @type         = opts[:type]
-        @group        = opts[:group] || {}
-        @dependent_on = opts[:dependent_on]
+        @element          = opts[:element]
+        @rename           = opts[:rename]
```

Comment:
Rename is a strange variable since it's a verb. How about just `name`?

Reply:
Yeah, but `name` is ambiguous with `element` because `element` holds the actual name of the param scope container name. So let's call it `element_renamed`, this causes less confusion I would argue.